<ui version="4.0" >
 <class>ItemsEditDialog</class>
 <widget class="QDialog" name="ItemsEditDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>257</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>617</width>
    <height>257</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>617</width>
    <height>257</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Edit Land</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="tile_edit_qt.qrc" >
    <normaloff>:/newPrefix/nel.png</normaloff>:/newPrefix/nel.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>220</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="availableItemsListWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QListWidget" name="selectedItemsListWidget" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>30</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addToItemsPushButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>90</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="tile_edit_qt.qrc" >
     <normaloff>:/newPrefix/right.png</normaloff>:/newPrefix/right.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Available Items:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Selected Items for:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFromItemsPushButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset resource="tile_edit_qt.qrc" >
     <normaloff>:/newPrefix/left.png</normaloff>:/newPrefix/left.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="itemNameLabel" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <includes/>
 <resources>
  <include location="tile_edit_qt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemsEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemsEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
